src/core/tests/type_prop/is_inf.cpp

```diff
@@ -0,0 +1,141 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "gtest/gtest.h"
+#include "openvino/opsets/opset10.hpp"
+#include "util/type_prop.hpp"
+
+using namespace std;
+using namespace ov;
+using namespace ov::opset10;
+
+TEST(type_prop, is_inf_default) {
+    const auto data = make_shared<Parameter>(element::f32, PartialShape{1, 64, 256, 256});
+    const auto is_inf = make_shared<IsInf>(data, IsInf::Attributes{});
```

Comment:
I think it would be more convinient to use the `IsInf` op with default attributes like `IsInf(data)` instead of `IsInf(data, IsInf::Attributes{})`, additional constructor should be considered.
```suggestion
    const auto is_inf = make_shared<IsInf>(data);
```

Reply:
Done